src/Temporalio/Client/Configuration/ClientConfig.cs

```diff
@@ -0,0 +1,124 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+
+namespace Temporalio.Client.Configuration
```

Comment:
```suggestion
namespace Temporalio.Client.EnvConfig
```

I think `EnvConfig` is better because 1) that's what we're calling it everywhere else, and 2) there's a lot of configuration not in here, so it's not really general purpose "configuration"

Reply:
Renamed